flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/GenerateUtils.scala

```diff
@@ -909,6 +908,62 @@ object GenerateUtils {
            |}
          """.stripMargin
       compares += code
+    })
+    compares.mkString
+  }
+
+  /**
+   * Generates code for comparing row keys.
+   * TODO: Remove this after [[MultiFieldRangeBoundComparatorCodeGenerator]]
+   * is refactored using SortSpec.
+   */
+  def generateRowCompare(
```

Comment:
It seems this method is not used. Is it introduced by accident?

Reply:
Weird, I did not add that. Will revert.

Reply:
@wuchong Did you rebase off master? Diffing master I see this was added.